_maps/map_files/RandomZLevels/beach.dmm

```diff
@@ -19,7 +19,6 @@
 /turf/simulated/floor/beach/away/water/deep/sand_floor,
 /area/awaymission/undersea)
 "af" = (
-/obj/structure/constructshell,
```

Comment:
Please mapmerge this properly instead of making a manual map edit (key `"ah"` is the exact same as this).

Reply:
map merged